test/parse-error-test/annotation_semantics/parse-error.expected

```diff
@@ -0,0 +1,4 @@
+[error] test/parse-error-test/annotation_semantics/input.ddl[3.4] failure: string matching regex `\p{javaJavaIdentifierStart}\p{javaJavaIdentifierPart}*' expected but `)' found
```

Comment:
I didn't get `Unrecognized semantics` error using the code above. Can you give a pointer for `^?`? I'll take a closer look.